src/js/README.md

```diff
@@ -1,12 +1,40 @@
-# js/pyodide
+# Pyodide Javascript package
 
-The Javascript code in this folder is responsible for the following tasks:
+<a href="https://www.npmjs.com/package/pyodide"><img src="https://img.shields.io/npm/v/pyodide" alt="npm"></a>
 
-1. Defines the public Javascript API
+## Usage
+
+Download and extract Pyodide packages from [Github
+releases](https://github.com/pyodide/pyodide/releases)
+(`pyodide-build-*.tar.bz2`). The version of the release needs to match exactly the version of this package.
```

Comment:
We should try to simplify this by copying the `build` directory into `src/js` in the `prepublish` script.

Reply:
I agree that we need to simplify, but as discussed in https://github.com/pyodide/pyodide/issues/14#issuecomment-875116949 I'm not convinced that bundling all packages is the solution. We need a solution that would also work for micropip installed packages anyway. 

So I'm planning to add some local caching next and support remote CDN in node so there is no difference between browser and node usage when starting Pyodide.